curriculum/challenges/english/16-the-odin-project/top-learn-variables-and-operators/learn-variables-and-operators-question-b.md

```diff
@@ -0,0 +1,64 @@
+---
+id: 65e18d61500d930ce8ed90a5
+title: Learn Variables and Operators Question B
+challengeType: 15
+dashedName: learn-variables-and-operators-question-b
+---
+# --description--
+
+You can think of variables as "storage containers" for data in your code.
+
+<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/d39eaf2ca95e80705f703bb218216c10508f5047/foundations/javascript_basics/fundamentals-1/imgs/00.png" alt='the concept of variables in coding as boxes labeled "birthday" that store and provide access to data, specifically a date "16-09-2003"'>
```

Comment:
should this point to our cdn link instead?

Reply:
I was planning on doing that after